docs/project.md

```diff
@@ -167,6 +167,8 @@ HEMTT will use the specified mod name (without `@`) to form `@mod` folder. Suppo
 
 HEMTT will use the specified key name for `.bikey` and `.biprivatekey` names. Supports [templating](/templating.md).
 
+The default is `{{prefix}}_{{version}}`, unless `reuse_private_key` is `true`, in which case the default is set to `{{prefix}}`
```

Comment:
It would be nice if this had a table instead, showing the difference between `reuse_private_key` being true or false. This would help clearly show what the default is, and what the output will be in a clear and readable way.